### PR TITLE
Change layout of feedback

### PR DESCRIPTION
**Pull request by oterral**
For #3766 

[Test](https://mf-geoadmin3.int.bgdi.ch/fix_3766/index.html?topic=ech&lang=fr&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bav.haltestellen-oev,ch.swisstopo.swisstlm3d-wanderwege&layers_visibility=false,false,false,false&layers_timestamp=18641231,,,)

### CODE REVIEW

**Comment by davidoesch** [comment]
As discussed: p1d1d1 will reduce the text in last paragraph to "Mit dem Absenden Ihre Meldung erklären Sie sich mit den  Nutzungsbedingungen einverstanden" and organize the entry on the Impressum and send the anchor link

**Comment by p1d1d1** [comment]
**Text in last paragraph:**
DE
Mit dem Absenden Ihre Meldung erklären Sie sich mit den [Nutzungsbedingungen ](https://www.geo.admin.ch/de/ueber-geo-admin/impressum.html#feedback)einverstanden
FR
En envoyant votre message, vous acceptez les [Conditions d'utilisation](https://www.geo.admin.ch/fr/geo-admin-ch/impressum.html#feedback)
IT
Inviando un messaggio, l’utente accetta le [Condizioni di utilizzo](https://www.geo.admin.ch/it/geo-admin-ch/colophon.html#feedback)
EN
By submitting your message, you agree to the [Terms of use](https://www.geo.admin.ch/en/about-swiss-geoportal/responsabilities-and-contacts.html#feedback)
RM
[Cundiziun d'utilisaziun](https://www.geo.admin.ch/en/about-swiss-geoportal/responsabilities-and-contacts.html#feedback)

**Drawing**
Text should be disabled. Only point an line

**General**
add as in mockup
1. ...
2. ...
3. ...

**Rumantsch**
@davidoesch we're missing RM. I suggest to use existing as follows:

1. Rapport charta actual (necessari)
2. Voss dissegn
3. Voss e-mail
Agiuntar datoteca

**Comment by davidoesch** [comment]
RM: correct

**Comment by p1d1d1** [comment]
Any further step here?

File: src/locales/rm.json [hunk 1]
```diff
@@ -164,16 +164,16 @@
 "featuretree_tooltip": "Tschertga tut ils objects in rectangul",
 "feedback_attachment": "Agiuntar datoteca (opziunal)",
 "feedback_create_geometry": "Vus pudais agiuntar in dissegn a Voss rapport: ",
-"feedback_description": "Rapport charta actual",
-"feedback_drawing": "Voss dissegn (opziunal)",
-"feedback_error_message": "Problem: rapport na tramess cun success",
+"feedback_description": "",
+"feedback_disclaimer": " <a href='https://www.geo.admin.ch/rm/about-swiss-geoportal/impressum.html#drawing' target='_blank'>Cundiziuns d'utilisaziun</a>",
```
Comment (loicgasser) [inline src/locales/rm.json]: @davidoesch text is incomplete in rm

File: src/locales/en.json [hunk 2]
```diff
@@ -331,14 +331,13 @@
 "query_custom": "Custom filter",
 "query_duplicate": "Duplicate condition",
 "query_layer": "--- Select a layer ---",
-"query_more_values": "Retrieve more values",
```
Comment (loicgasser) [inline src/locales/en.json]: still used in:

src/components/query/partials/query.html

no?

**Comment by p1d1d1** [comment]
RM is here: https://github.com/geoadmin/mf-geoadmin3/pull/3778#issuecomment-295289376 end of the comment

**Comment by p1d1d1** [comment]
@oterral @gjn update here? thanks

File: src/locales/en.json (hunk 2 above)
Comment (oterral) [inline src/locales/en.json]: yes good found

**Comment by oterral** [comment]
@p1d1d1 done.

**Comment by p1d1d1** [comment]
@oterral many thanks. Any chance to have an updated test link?

**Comment by oterral** [comment]
The link is the same it works for me

**Comment by p1d1d1** [comment]
Ok sorry. Cache issue.

**Comment by p1d1d1** [comment]
@oterral one quick fix. The text (optional) in "Attach a file (optional)" should be deleted, thanks.
All languages.

**Comment by p1d1d1** [comment]
@oterral @gjn nice job, many thanks.

**Comment by davidoesch** [comment]
on mobiel we don't have  numer 2. drawing

proposal: we remove the 1.2.3 in the text.

other proposals?

image

**Comment by p1d1d1** [comment]
We invert 2 and 3. First e-mail, then drawing

**Comment by davidoesch** [comment]
As discussed earlier: we had this once before.... But give it a try

**Comment by gjn** [comment]
Please open new issue as this will get lost otherwise.

**Comment by davidoesch** [comment]
ping @p1d1d1

**Comment by p1d1d1** [comment]
as soon as we agree how to proceed

**Comment by p1d1d1** [comment]
Not action. So has been decided.

**Comment by oterral** [comment]
So we don't invert 2 and 3 ?

**Comment by p1d1d1** [comment]
No.

**Pull request merged**